packages/e2e-cypress/package.json

```diff
@@ -44,6 +44,7 @@
   },
   "devDependencies": {
     "@types/lodash": "^4.14.178",
+    "coverage-istanbul-loader": "^3.0.5",
```

Comment:
Remember that the Cypress AE devDeps are the ones which are used into dev phase of the app extension, while packages which must be used into userland project must go into the app extension normal dependencies

So this package should be moved to `dependencies`

Reply:
Okay, I fixed it.